src/cr/cube/cube.py

```diff
@@ -516,9 +539,14 @@ def _slice_idxs(self):
 
     @lazyproperty
     def _valid_idxs(self):
-        return np.ix_(
+        valid_idxs = np.ix_(
             *tuple(d.valid_elements.element_idxs for d in self._all_dimensions)
         )
+        return (
+            valid_idxs[::-1]
```

Comment:
This is going to get more elaborate in the future, probably. We'll need a way to tell to the cube _which_ dimensions we want transposed and which not. Just keep it in mind.

Reply:
ok i'll write a todo there